include/rocksdb/comparator.h

```diff
@@ -20,7 +21,7 @@ class Slice;
 // used as keys in an sstable or a database.  A Comparator implementation
 // must be thread-safe since rocksdb may invoke its methods concurrently
 // from multiple threads.
-class Comparator {
+class Comparator : public Customizable {
```

Comment:
As this is public interface, is there potential ABI breakage?

Reply:
I am not sure what the ABI breakage would be or how to test for it. Ideas?

Reply:
I think it's fine to break binary compatibility, especially for non patch release. I found a website which tracks our API binary compatibility: https://abi-laboratory.pro/?view=timeline&l=rocksdb